src/api/verify.ts

```diff
@@ -16,13 +17,15 @@ let expectedCountryCode: string | null = null;
 let expectedStateCode: string | null = null;
 
 class VerifyAPI {
-  static async trackVerified(params: RadarTrackParams, encrypted: Boolean = false) {
+  static async trackVerified(params: RadarTrackVerifiedParams, encrypted: Boolean = false) {
     const options = Config.get();
 
+    const { skipVerifyApp } = params;
+
     // user indentification fields
     const userId = params.userId || Storage.getItem(Storage.USER_ID);
-    const deviceId = params.deviceId || Device.getDeviceId();
-    const installId = params.installId || Device.getInstallId();
```

Comment:
Why the change to get rid of setting these via params? This might be how we override the values int the simulator.

Reply:
I don't really want to give developers control of these for fraud / geo-compliance use cases. For implementations using the Mac and Windows app, those apps override these anyway (with more stable hardware device IDs)